kernelci/config/build.py

```diff
@@ -49,6 +49,33 @@ def url(self):
         return self._url
 
 
+class Reference(YAMLObject):
+    """Kernel git tree model."""
```

Comment:
Ah I think this docstring was copied from the Tree class.

Reply:
Corrected